src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -247,6 +272,44 @@ public synchronized ProcessWrapper launchTranscode(
 		return pw;
 	}
 
+	/**
```

Comment:
I agree that doesn't use the same code twice will be good.

But, just to clear, when i do the changes asked and that errors arise, i just stop brutally to be interested...
So, or you push a commit, or this PR will wait until my mood become better...